### DISCUSSION TITLE
How to set "includePath" search the current directory first?

### DISCUSSION THREAD

**Discussion by Lwxiang** [microsoft/vscode-cpptools#9625] [Q&A]
in `c_cpp_properties.json`, I can set
```json
{
    "configurations": [
           {
                   "includePath": [
                            "${workspaceFolder}/**",
                            "/usr/local/include/**"
                   ]
           }
     ]
}
```

My header files in `${workspaceFolder}` can be searched, but it seems like always with an order by file tree.
How to set the `includePath` to search current directory first? Like `g++ -I./`

**Comment by sean-mcmanus** [comment]
Using `#include ""` instead of `#include <>` should use the current directory first. Otherwise, there isn't a way to set the includePath property like that. Other include path configurations options such as compileCommands or configurationProvider are able to provide per-TU include path settings.
